substrate/frame/scheduler/src/migration.rs

```diff
@@ -19,7 +19,6 @@
 
 use super::*;
 use frame_support::traits::OnRuntimeUpgrade;
-use frame_system::pallet_prelude::BlockNumberFor;
 
 #[cfg(feature = "try-runtime")]
 use sp_runtime::TryRuntimeError;
```

Comment:
We should probably deprecate the migrations. They would not interact so well with changing the BN provider at the same time and it is a lot of legacy code